clerk/tokens.go

```diff
@@ -46,8 +47,18 @@ func (c *client) DecodeToken(token string) (*TokenClaims, error) {
 	return &TokenClaims{Claims: standardClaims, Extra: extraClaims}, nil
 }
 
+type verifyTokenOptions struct {
+	authorizedParties []string
```

Comment:
Can we make this a map so that we avoid the linear search?

Reply:
Nice catch, changed in [10dc7fa](https://github.com/clerkinc/clerk-sdk-go/pull/43/commits/10dc7fa305c1b2fffa2993a37f93b1f92461ece1)